web/public/causality.mjs

```diff
@@ -0,0 +1,49 @@
+import "./d3.v355.min.js";
+import './graphviz.min.js';
+
+export function renderCausality(dot){
+  const foundSvg = d3.select(".causality-graph");
+  const svg = createSvg(foundSvg)
+
+  graphviz.graphviz.layout(dot, "svg", "dot").then(content => {
+    svg.html(content);
+
+    svg.selectAll('title').remove();
+    svg.selectAll('*').attr('xlink:title', null);
```

Comment:
Could we add a comment explaining why we need this? IIRC it was to get rid of the HTML tooltips that can't be disabled with graphviz